### PR TITLE
[KYUUBI #2576][FOLLOWUP] Bump Hudi 0.11.0

### PR DESCRIPTION
**Pull request by cxzl25**
### _Why are the changes needed?_

- Resolve WARNING prompt for IntelliJ IDEA Build Panel
- Re-adjusted the order of several hudi moudles to make it consistent in the order of server and engine

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cxzl25** [comment]
IntelliJ IDEA Build Panel has this WARNING prompt after #2576 was merged. 
Thanks for reporting the problem to me. @pan3793 

image

The problem is introduced by hbase related dependencies.

In the research and debugging of IntelliJ IDEA `RemoteMavenServer` and `maven-dependency-plugin`, it is found that it is caused by specifying a range of dependent versions.

```xml
        <dependency>
            <groupId>org.glassfish</groupId>
            <artifactId>javax.el</artifactId>
            <version>[3.0.0,)</version>
        </dependency>
```

https://repo1.maven.org/maven2/org/glassfish/javax.el/3.0.0/javax.el-3.0.0.pom 
(repository jvnet-nexus-release)


maven-resolver-provider first obtains the pom through 3.0.0, then obtains the repository described by the pom, and finally through the central repo and the repository described by the pom, summarizes the version list, and then obtains the latest version.
Because several versions corresponding to the repository described by pom (https://maven.java.net/content/repositories/snapshots/org/glassfish/javax.el/maven-metadata.xml) do not exist, such a prompt appears.


REF: https://issues.apache.org/jira/browse/HBASE-18831

**Comment by pan3793** [comment]
Thanks for digging the root cause, I verified the PR locally and it works!

**Review by pan3793 — APPROVED** [review]

File: kyuubi-server/pom.xml [hunk 1]
```diff
@@ -387,6 +387,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-spark3.1.x_${scala.binary.version}</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+
```
Comment (pan3793) [inline kyuubi-server/pom.xml:396]: nit: one empty line

**Comment by pan3793** [comment]
Thanks, merging to master

**Pull request closed without merging**